CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Packer Template Changelog
```

Comment:
template?

Reply:
Changed all occurrences to _configuration_